<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DDC2-L-F2DF37H" name="Run All Tests" tests="1" failures="1" timestamp="7 Dec 2020 14:15:46 GMT" time="2.285" errors="0">
  <testcase name="feature" time="1.946" classname="main.CucumberRunner">
    <failure type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: expected [200] but found [401]">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: expected [200] but found [401]
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.AssertionError: expected [200] but found [401]
	at stepMethods.APIStepMethods.launchAPI(APIStepMethods.java:21)
	at stepDefinition.APIStepDefinition.userLaunchesTheAPIWith(APIStepDefinition.java:13)
	at âœ½.Given user launches the API with "Kanpur"(Phase2.feature:5)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- Run All Tests -->
